lib/install/data/data.go

```diff
@@ -263,5 +263,7 @@ func (d *Data) CopyNonEmpty(src *Data) error {
 
 	d.Timeout = src.Timeout
 
+	d.DNS = src.DNS
```

Comment:
Question - should this be checking that `src.DNS.IsSet == true` before copying to `d.DNS`?